app.rb

```diff
@@ -0,0 +1,58 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_student(age, name, parent_permission)
+    classroom = 'Null'
+    @people << { type: 'Student', data: Student.new(age, name, classroom, parent_permission) }
+  end
+
```

Comment:
- [ ] Great job but I think you have an error hereyou are passing a  wrong number of arguments (given 4, expected 1..3)  so when I am try to create a new student I can't ceate it because there is not coordinates with the given arguments so please fix this error to let the user be able to create a new student 💯😊📌🔥♻✅

## this is the given error ♻📌🔥💯💯
![image](https://user-images.githubusercontent.com/58553711/216479619-03da77f4-7128-4569-bc1c-a627a906dce5.png)